tests/example.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@playwright/test';
```

Comment:
do we need this file?

Reply:
I will add the demo files to .gitignore - they are good to have for reference but do not need to live here.